common/lib/xmodule/xmodule/video_module/video_module.py

```diff
@@ -304,12 +304,9 @@ def __init__(self, *args, **kwargs):
             self._field_data.set_many(self, field_data)
             del self.data
 
-        editable_fields = super(VideoDescriptor, self).editable_metadata_fields
```

Comment:
@ormsbee The editable_metadata_fields is a property on xblock that returns fields with scope `Scope.settings`, which can be edited in Studio.

Inspired by @cpennington's suggestion "it looks to me like it's only using [editable_metadata_fields] data to find out if specific fields are explicitly set, which could just be done w/ self.runtime.get_field_provenance, which might skip a lot of the work", I replaced the `editable_metadata_fields` call with the calls below to check whether a field is explicitly set.

So the overall behavior of the `__init__` method has remained the same.  What's changed is how it determines whether a field is explicitly set.